src/Components/Endpoints/src/RazorComponentEndpointInvoker.cs

```diff
@@ -33,6 +34,9 @@ public Task Render(HttpContext context)
         return _renderer.Dispatcher.InvokeAsync(() => RenderComponentCore(context));
     }
 
+    // We do not want the debugger to consider NavigationExceptions caught by this method as user unhandled.
+    [MethodImpl(MethodImplOptions.NoInlining)]
```

Comment:
From the API review, I thought the conclusion was that the JIT team thought that it would never inline a method with a catch block, so these shouldn't be necessary, and if they are, we should teach the JIT about `DebuggerDisableUserUnhandledExceptions`

Reply:
I removed all the instances `[MethodImpl(MethodImplOptions.NoInlining)]`.